lib/charms/grafana_k8s/v0/grafana_dashboard.py

```diff
@@ -115,15 +145,22 @@ def __init__(
                 for dashboard validity. When events on `event_relation`
                 occur, this consumer library will invalidate or
                 restore the dashboard
+            dashboards_path: a filesystem path relative to the charm root
+                where dashboard templates can be located
         """
         super().__init__(charm, name)
         self.charm = charm
         self.name = name
+        self._DASHBOARDS_PATH = dashboards_path
         self._stored.set_default(dashboards={}, dashboard_templates={}, event_relation=None)
         self._stored.event_relation = event_relation
```

Comment:
Duplicated assignment
```suggestion
        self._stored.set_default(dashboards={}, dashboard_templates={}, event_relation=event_relation)
```